ruleset/sca/darwin/20/cis_apple_macOS_11.01.yml

```diff
@@ -0,0 +1,914 @@
+    condition: all
+    rules:
+      - 'c:defaults read /Library/Preferences/com.apple.loginwindow SHOWFULLNAME -> 1'
+
+# jal 
+# 6.1.2 Disable "Show password hints" (Automated) 
+  - id: 18063
+    title: "Disable \"Show password hints\""
+    description: "Password hints are user created text displayed when an incorrect password is used for an account."
+    rationale: "Password hints make it easier for unauthorized persons to gain access to systems by providing information to anyone that the user provided to assist remembering the password. This info could include the password itself or other information that might be readily discerned with basic knowledge of the end user."
+    remediation: "Run the following command to disable password hints: sudo defaults write /Library/Preferences/com.apple.loginwindow RetriesUntilHint -int 0"
+    compliance:
+      - cis: ["6.1.2"]
+    condition: any
+    rules:
+      - 'c:defaults read /Library/Preferences/com.apple.loginwindow RetriesUntilHint -> 0'
+      - 'c:defaults read /Library/Preferences/com.apple.loginwindow RetriesUntilHint -> r:does not exist'
+
+# jal
+# 6.1.3 Disable guest account login (Automated)
+  - id: 18064 
+    title: "Disable guest account login"
+    description: "The guest account allows users access to the system without having to create an account or password. Guest users are unable to make setting changes, cannot remotely login to the system and all created files, caches, and passwords are deleted upon logging out."
+    rationale: "Disabling the guest account mitigates the risk of an untrusted user doing basic reconnaissance and possibly using privilege escalation attacks to take control of the system."
+    remediation: "Run the following command in Terminal: sudo defaults write /Library/Preferences/com.apple.loginwindow GuestEnabled - bool false"
+    compliance:
+      - cis: ["6.1.3"]
+    condition: all
+    rules:
+      - 'c:defaults read /Library/Preferences/com.apple.loginwindow.plist GuestEnabled -> 0'
+
+# jal 
+# 6.1.4 Disable "Allow guests to connect to shared folders" (Automated)
+  - id: 18065 
+    title: "Disable \"Allow guests to connect to shared folders\""
+    description: "Allowing guests to connect to shared folders enables users to access selected shared folders and their contents from different computers on a network."
+    rationale: "Not allowing guests to connect to shared folders mitigates the risk of an untrusted user doing basic reconnaissance and possibly use privilege escalation attacks to take control of the system."
+    remediation: "Run the following command in Terminal: sudo defaults write /Library/Preferences/com.apple.loginwindow GuestEnabled - bool false"
+    compliance:
+      - cis: ["6.1.4"]
+    condition: any
+    rules:
+      - 'c:defaults read /Library/Preferences/com.apple.AppleFileServer guestAccess -> 0'
+      - 'defaults read /Library/Preferences/SystemConfiguration/com.apple.smb.server AllowGuestAccess -> 0'
```

Comment:
To avoid this error:
```
2020/12/31 11:00:38 sca: ERROR: Invalid rule: 'defaults read /Library/Preferences/SystemConfiguration/com.apple.smb.server AllowGuestAccess -> 0'. Skipping policy.
2020/12/31 11:00:38 sca: ERROR: Error while evaluating the policy '/Library/Ossec/ruleset/sca/cis_apple_macOS_11.01.yml'
```
this check is missing the type prefix, it must start by `c:`. Same for the check two lines below.